src/translations/th.json

```diff
@@ -0,0 +1,293 @@
+                "analyticsHintP2": "มันจะช่วยให้เราพัฒนาได้ดีขึ้น เราขอร้องให้คุณเปิดฟังก์ชันนี้",
+                "stealFocus": "ดึงความสนใจจากแถบ Url",
+                "stealFocusHint": "ถ้าเปิดใช้งาน ระบบจะบังคับให้ Browser สนใจ Anori แทนที่จะเป็นแถบ Url ด้านบนทำให้สามารถใช้เมนูคำสั่งได้ (Cmd + K) โดยที่ไม่ต้องกดที่หน้าอีกครั้ง",
+                "automaticCompact": "เปลี่ยนเป็นโหมดกระทัดรัดโดยอัตโนมัติ (ตามขนาดหน้าจอ)",
+                "automaticCompactModeThreshold": "เปลี่ยนเป็นโหมดกระทัดรัดถ้าขนาดหน้าจอเล็กกว่า",
+                "automaticCompactModeThresholdHint": "ความกว้างหน้าจอของคุณตอนนี้คือ {{screenWidth}}px",
+                "useCompact": "ใช้โหมดกระทัดรัดตลอดเวลา",
+                "showAnimationOnOpen": "ใช้งานแอนิเมชั่นเมื่อเปิด",
+                "showAnimationOnOpenHint": "ถ้าเปิดใช้งาน Anori จะมีแอนิเมชั่นในการโหลดหน้าซึ่งจะซ่อนสิ่งต่าง ๆ ที่ยังโหลดไม่เสร็จและจะดูสวยงามกว่าการปิด",
+                "hideEditButton": "ซ่อนปุ่ม 'แก้ไขโฟลเดอร์'",
+                "hideEditButtonHint": "เรียบง่ายเป็นพิเศษ เย้"
+            },
+            "customIcons": {
+                "title": "ไอคอนที่กำหนดเอง",
+                "incorrectFormat": "กรุณาใช้ไฟล์ที่ถูกต้อง Anori รองรับไฟล์ .png, .jpg, .gif, .jpeg และ .svg",
+                "noIcons": "คุณไม่มีไอคอนที่กำหนดเอง",
+                "iconName": "ชื่อไอคอน",
+                "invalidNames": "ไอคอนบางอันมีชื่อที่ไม่ถูกต้อง กรุณาแก้ไขก่อนการบันทึก",
+                "nameContainsInvalidChars": "ชื่อไอคอนมีตัวแักษรที่ไม่ถูกต้อง รองรับเฉพาะ ตัวอักษร, ตัสเลข, - และ  _ เท่านั้น",
+                "saveIcons": "บันทึกไอคอน",
+                "supportedFormats": "Anori รองรับ .png, .jpg, .gif และ .svg. แนะนำให้ใช้รูปสี่เหลี่ยมกว้างระหว่าง 128 และ 256 พิกเซล (ไม่มีผลกับ .svg).",
+                "importIcons": "นำเข้าไอคอน"
+            },
+            "folders": {
+                "title": "โฟลเดอร์",
+                "createNew": "สร้างโฟลเดอร์ใหม่",
+                "defaultName": "โฟลเดอร์ใหม่"
+            },
+            "pluginSettings": {
+                "title": "ตั่งค่าส่วนเสริม"
+            },
+            "theme": {
+                "title": "ธีม"
+            },
+            "importExport": {
+                "title": "นำเข้า และ ส่งออก",
+                "info": "คุณสามารถส่งออกการตั่งค่าและนำเข้าอันเก่าได้",
+                "import": "นำเข้า",
+                "export": "ส่งออก"
+            },
+            "aboutHelp": {
+                "title": "ความช่วยเหลือ และ เกี่ยวกับส่วนเสริมนี้",
+                "onlyEnglishPlease": "ฉัน (ผู้เริ่มโครงการ) พูดได้ไม่กี่ภาษาดังนั้นกรุณาแจ้งปัญหาเป็นภาษาอังกฤษ ขอบคุณ!",
+                "p1": "ส่วนเสริม Anori ฟรีและ open source, Source code สามารถดูได้ที่ <0>GitHub</0>, ถ้าคุณต้องการเสนอฟังก์ชันเพิ่มเติมหรือรายงานปัญหา กรุณาสร้าง <1>new issue</1> ใน Github หรือถ้าแค่อยากขอบคุณ คุณสามารถกดดาวให้กับโครงการของเราได้ :).",
```

Comment:
Here (and later in `p3`) could you translate text inside `<1></1>`? This will be link caption, so it will look weird untranslated in the middle of Thai text